DigitalLearningSolutions.Data/Models/Centres/Centre.cs

```diff
@@ -31,8 +31,11 @@ public class Centre
         public int TrainerSpots { get; set; }
         public string? IpPrefix { get; set; }
         public string? ContractType { get; set; }
+        public int CentreTypeId { get; set; }

```

Comment:
These three new fields are no longer used right?

Reply:
If that's correct, you can go ahead and merge after they're gone.

Reply:
Ah yes, I missed those. I've removed them now.